CMakeLists.txt

```diff
@@ -316,22 +329,18 @@ set(CPACK_PACKAGE_FILE_NAME "${PROJECT_NAME}_${CPACK_PACKAGE_VERSION}_${PACKAGE_
 include(CPack)
 
 if (UNIX AND NOT MINGW AND NOT APPLE)
-    install(TARGETS ${JA2_BINARY} RUNTIME DESTINATION bin)
+	install(TARGETS ${JA2_BINARY} RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
     if(BUILD_LAUNCHER)
-        install(TARGETS ${LAUNCHER_BINARY} RUNTIME DESTINATION bin)
+        install(TARGETS ${LAUNCHER_BINARY} RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
     endif()
-    install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/assets/externalized assets/mods assets/unittests DESTINATION share/ja2)
-    install(FILES ${STRACCIATELLA_SHARED_LIB} DESTINATION ${INSTALL_LIB_DIR})
-    install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/assets/distr-files-linux/ja2-stracciatella.desktop DESTINATION share/applications)
+	install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/assets/externalized assets/mods assets/unittests DESTINATION ${CMAKE_INSTALL_DATADIR}/ja2)
+    install(FILES ${STRACCIATELLA_SHARED_LIB} DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
if you won't be using INSTALL_LIB_DIR, why even set it?

Reply:
It is used, see https://github.com/hasufell/ja2-stracciatella/blob/f4635ad2cf67a2b5611c09a1f1c8b44a1e5d0327/CMakeLists.txt#L80

Reply:
In retrospect it's not clear if that's needed, since none of our code uses it. But more importantly, anyone relying on overriding INSTALL_LIB_DIR will now get files in a potentially different path, since there is no graceful migration set up.

Reply:
I don't see a problem with that since INSTALL_LIB_DIR is not documented here: https://github.com/ja2-stracciatella/ja2-stracciatella/blob/master/COMPILATION.md#additional-options